src/main/java/strategy/impl/OperationStrategyImpl.java

```diff
@@ -0,0 +1,20 @@
+package strategy.impl;
+
+import java.util.Map;
+import model.FruitTransaction;
+import strategy.OperationHandler;
+import strategy.OperationStrategy;
+
+public class OperationStrategyImpl implements OperationStrategy {
```

Comment:
The class should not begin with an empty line. Please remove the empty line to follow the coding standards.